mods/ts/rules/shared-support.yaml

```diff
@@ -39,3 +42,31 @@ NAPULS:
 		FactionImages:
 			gdi: napuls.gdi
 			nod: napuls.nod
+	ProvidesPrerequisite@gdi:
+		Prerequisite: emp.gdi
+		Factions: gdi
+		ResetOnOwnerChange: true
+	ProvidesPrerequisite@nod:
+		Prerequisite: emp.nod
+		Factions: nod
+		ResetOnOwnerChange: true
+	AttackOrderPower@nod:
+		Cursor: emp
+		Icon: emp
+		ChargeTime: 135
+		Description: E.M. Pulse
+		LongDesc: Fires a pulse blast which disables\nall mechanical units in the area.
+		EndChargeSound: speech-nod|00-i158.aud
```

Comment:
> 14:51:35 (abcdefg30) it's cause we're not referencing something out of notifications.yaml
14:51:40 (pchote) yeah
14:51:41 (abcdefg30) but the .aud file directly
14:52:04 (abcdefg30) sry, but fixing ^ is out of scope for that pr I think
14:53:03 (pchote) are you saying that we have never supported faction specific notifications for support powers?
14:53:06 (pchote) even with d2k?
14:53:10 (pchote) because that would be super bogus
14:53:21 (pchote) although I wouldn't be too surprised